CONTRIBUTING.md

```diff
@@ -1,3 +1,54 @@
 # SMI Services Contributing Guidelines
 
--   TODO 😊
+The SMI Services project team uses the Feature Branch Workflow to accomplish work in a consistent and productive manner. It is a Git workflow that uses feature branches to isolate new features and bug fixes from the main codebase, allowing multiple developers to work on the same codebase without causing conflicts or disrupting the main codebase.
+
+> **Feature Branch Workflow**: Clone Project, Create Branch, Add/Commit Changes, Push Changes, Open Pull Request, Request Review, Merge Changes, Repeat Process.
+
+Here is an example of a typical Feature Branch Workflow whilst working on the SMI project:
+
+1. **Clone Project**: As an individual developer (contributor) clone the central repository onto your local machine.
+
+    ```console
+    git clone git@github.com:SMI/SmiServices.git
+    ```
+
+2. **Create Branch**: For each new feature or bug fix, create a new branch from the main branch (usually called “master”) which will allow you to work on your own version of the code without affecting the main codebase.
+
+    ```console
+    git checkout -b new-feature
+    ```
+
+    This creates a new branch called "new-feature" and switches to that branch.
+
+3. **Add/Commit Changes**: You can make changes, add new code, and commit those changes to your local repository.
+
+    ```console
+    git add . 
```

Comment:
I'd like to discourage the use of `git add .`, as it may result in unwanted files being committed. A better workflow is like `git add -u` (to stage changes to existing files) and `git add file1 file2 ...` to add new files.